core/src/main/scala/caliban/schema/Types.scala

```diff
@@ -7,9 +7,6 @@ import scala.annotation.tailrec
 
 object Types {
 
-  /**
```

Comment:
Any reason to remove this?

Reply:
I deleted the extra method but didn't realise I also deleted the scaladoc 😅